lib/typings/types.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+ /* tslint:disable */
```

Comment:
I wouldn't recommend calling this file `types.ts`as it will get overwritten the second somebody puts in some graphql queries and runs the build.

Reply:
I'm not sure I understand - this is the autogenerated file from the schema in the pack.  Is there a way to change that auto-generation process to create a different file?

Reply:
After having a discussion with dd regarding the autofix fighting type generation, looks like I shouldn't even be committing the types file and letting it get generated during the build.  I'll push another commit removing it.